test/extended/networking/livemigration.go

```diff
@@ -128,6 +130,12 @@ var _ = Describe("[sig-network][OCPFeatureGate:PersistentIPsForVirtualization][F
 							vmCreationParams.NetworkName = nadName
 						}
 
+						if netConfig.preconfiguredIP != "" {
```

Comment:
I find it strange to see the static IP request (and MAC) and a network attribute. I think you're just doing it because it's easy. 

Can't we just add a new struct for workload specific configuration / customization params, and have that as variadic input of this function ? I think that aligns better with the resource model we have.

Reply:
> I find it strange to see the static IP request (and MAC) and a network attribute. I think you're just doing it because it's easy.

guilty as charged :) 

> Can't we just add a new struct for workload specific configuration / customization params, and have that as variadic input of this function ? I think that aligns better with the resource model we have.

So, if we're adding a new WorkloadNetworkConfig struct, might as well add the other workload related params {vmResource,opCmd}..